backend/infrahub/events/models.py

```diff
@@ -27,7 +27,7 @@ class ParentEvent(BaseModel):
 
 
 class EventMeta(BaseModel):
-    branch: Branch | None = Field(default=None)
+    branch: Branch | None = Field(default=None, description="The branch on which originate this event")
```

Comment:
This branch attribute will actually be removed as this information now exists within the context of the meta data.